packages/services/service-common/src/index.ts

```diff
@@ -5,3 +5,4 @@ export * from './metrics';
 export * from './heartbeats';
```

Comment:
I assume we should remove `FastifyLoggerInstance` ?

Reply:
this's the type we're using for the `registerShutdown` `logger` arg to avoid erroring for type conflicts, as you've mentioned above, there's going to be no need for it when we ensure all services have been migrated so that we can safely change the type for it knowing every service that's using it is passing the right arg type

Reply:
Got it, see my comment above.